superset/db_engine_specs/redshift.py

```diff
@@ -14,14 +14,60 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+import re
+
+from flask_babel import gettext as __
+
 from superset.db_engine_specs.postgres import PostgresBaseEngineSpec
+from superset.errors import SupersetErrorType
+
+# Regular expressions to catch custom errors
+TEST_CONNECTION_ACCESS_DENIED_REGEX = re.compile(
+    'FATAL:  password authentication failed for user "(?P<username>.*?)"'
```

Comment:
Let's keep this tight, in case the level changes or the whitespace changes (those 2 spaces look suspicious).

```suggestion
    'password authentication failed for user "(?P<username>.*?)"'
```

Reply:
hi